### DISCUSSION TITLE
mirrormaker2-init CRB not cleaned up

### DISCUSSION THREAD

**Discussion by lenin-joseph** [strimzi/strimzi-kafka-operator#8966] [Q&A]
Hello,

Whenever MM2 is deployed with Rack configurations, it creates a mirrormaker2-init CRB. However, when we delete the kafkamirrormaker2 and uninstall the cluster-operator helm, the CRB is not getting cleaned up from the cluster.

We verified this for Kafka clusters, but didn't see any issues there and kafka-init CRBs are getting cleaned up.

Could you take a look and let us know if we are missing anything here or is it a BUG?

Thank you,
Lenin

**Comment by scholzj** [comment]
That looks like a bug. I will have a look at it. Thanks for raising this.

**Reply by scholzj** [reply]
I opened #8967 for it.
